console/consoleapi/sharing.go

```diff
@@ -0,0 +1,244 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleapi
```

Comment:
Why is it called console?

Reply:
@egonelbre so, i named this package as analog packages were named in satellite and storagenode.
This package contains our business logic (as in satellite and storagenode) and server + api + web pages

Reply:
Yeah, but `console` initially meant a "dashboard where you control all your things".

I think httpserver and handler names were fine.

Reply:
but now we have more than one handler, we have several handlers (http.Handler) for different endpoints
also i tried to make them as much atomic as possible, so i placed all business logic from transport layer
so basically we have service and server

Reply:
Un that case the name should reflect the domain, whatever it is. What are the handlers?

Reply:
when i said Handler - i meant function that will be called on appropriate http request
```
router.HandleFunc("/{serialized-access}/{bucket-name}", sharingController.BucketFiles).Methods(http.MethodGet)
	router.HandleFunc("/{serialized-access}/{bucket-name}/{file-name}", sharingController.File).Methods(http.MethodGet)
	router.HandleFunc("/raw/{serialized-access}/{bucket-name}/{file-name}", sharingController.OpenFile).Methods(http.MethodGet)
```

i totally agree that it should reflect domain name. Could you help me a bit with choosing better one please?

Reply:
Maybe `sharing` and `httpapi`?

Although, I really don't see a good reason to split them into separate packages at the moment.